src/main/java/org/spongepowered/mod/mixin/core/world/MixinWorld.java

```diff
@@ -58,10 +64,11 @@
 import org.spongepowered.mod.world.gen.SpongeGeneratorPopulator;
 
 import java.io.File;
+import java.util.List;
 
 
 @NonnullByDefault
-@Mixin(net.minecraft.world.World.class)
+@Mixin(value = net.minecraft.world.World.class, priority = 1111)
```

Comment:
I could. I just used the same value that's in [mixins.forge.core.json](https://github.com/SpongePowered/Sponge/blob/master/src/main/resources/mixins.forge.core.json#L6)